apps/commuter_rail_boarding/test/trip_cache_test.exs

```diff
@@ -28,12 +27,6 @@ defmodule TripCacheTest do
     end
 
     test "correctly finds a trip ID based on the date passed in" do
-      # find the next Saturday
-      day_of_week = @datetime |> DateHelpers.service_date() |> Date.day_of_week()
-
-      unix = DateTime.to_unix(@datetime)
-      unix_saturday = unix + 86_400 * (6 - day_of_week)
-      saturday = DateTime.from_unix!(unix_saturday)
```

Comment:
praise: thanks for cleaning this up!

Reply:
No problem! Always happy to fix warnings.